atmat/lattice/atfitchrom.m

```diff
@@ -25,7 +25,7 @@
 %
 % See also ATFITTUNE
 
-check_radiation(ring,false);
+%check_radiation(ring,false);
 [deltaP,varargin]=getoption(varargin,'DPStep');
 [deltaS,varargin]=getoption(varargin,'HStep',1.0e-5);
 [dpp,varargin]=getargs(varargin,0.0,'check',@(arg) isscalar(arg) && isnumeric(arg));
```

Comment:
It should be
```python
[dpp,varargin]=getargs(varargin,NaN,'check',@(arg) isscalar(arg) && isnumeric(arg));
```
to avoid warnings if dpp is provided in 6D

Reply:
Ah thanks! My matlab skills are very limited, would you have time to fix this? Also we found a bug in atenergy() that is still looking for the harmonic number on cavities.

We need these fixed for tomorrow for the restart....

Reply:
Can you handle atenergy() as well?